Pipfile

```diff
@@ -0,0 +1,23 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+
+[dev-packages]
+docutils = "*"
+"flake8" = "*"
+futures = "<3.2.0"
+isort = "*"
+modernize = "*"
+multilint = "*"
+numpy = "*"
+pytest = "*"
+six = "*"
+factory_boy = "*"
+Faker = "*"
+Pygments = "*"
+
+[requires]
+python_version = "2.7"
```

Comment:
locking to py 2.7 - will this really help with the optinoal dependencies?

Reply:
That only affects `pipenv check` command. This line can be deleted, I guess.